examples/tti_operators.py

```diff
@@ -90,11 +90,11 @@ def smart_sympy_replace(self, num_dim, time_order, res, funs, arrs, fw):
         for fun, arr in zip(funs, arrs):
             if num_dim == 2:
                 # Replace function notation with array notation
-                res = res.replace(fun(a, b, c), arr[a, b, c])
-                res = res.replace(arr[a*x+b, c*z+d, e*t+f], arr[e*t+f, a*x+b, c*z+d])
+                res = res.replace(fun(a, b, c), arr.indexed[a, b, c])
+                res = res.replace(arr.indexed[a*x+b, c*z+d, e*t+f], arr.indexed[e*t+f, a*x+b, c*z+d])
             else:  # num_dim == 3
-                res = res.replace(fun(a, b, c, d), arr[a, b, c, d])
-                res = res.replace(arr[x+b, y+q, z+d, t+f], arr[t+f, x+b, y+q, z+d])
+                res = res.replace(fun(a, b, c, d), arr.indexed[a, b, c, d])
+                res = res.replace(arr.indexed[x+b, y+q, z+d, t+f], arr.indexed[t+f, x+b, y+q, z+d])
```

Comment:
This is an example of where multiple calls to `indexed` result in multiple objects being manipulated in the same line which actually refer to the same term in the expression.